### PR TITLE
Add deafen button & change text buttons to icons in voice page

### PR DESCRIPTION
**Pull request by ryanalexander**

### CODE REVIEW

**Comment by ryanalexander** [comment]
Extra commits were to remove things what should not be in this pr

File: src/components/common/user/UserIcon.tsx [hunk 1]
```diff
@@ -12,12 +12,12 @@ import { AppContext, useClient } from "../../../context/revoltjs/RevoltClient";
...
 import IconBase, { IconBaseProps } from "../IconBase";
 import fallback from "../assets/user.png";
+import {VolumeMute} from "@styled-icons/boxicons-solid";
 
-type VoiceStatus = "muted";
 interface Props extends IconBaseProps<User> {
     mask?: string;
     status?: boolean;
-    voice?: VoiceStatus;
+    voice?: string;
```
Comment (insertish) [inline src/components/common/user/UserIcon.tsx:20]: Why is this type becoming ambiguous?
I don't see why it can't just be "muted" | "deaf".

**Review by insertish — CHANGES_REQUESTED** [review]
The rest of the code looks fine, I just don't like the type change here.

File: src/components/common/user/UserIcon.tsx (hunk 1 above)
Comment (ryanalexander) [reply src/components/common/user/UserIcon.tsx:20]: It was adding more complexity than needed (in my eyes).

The same functionality can be done with a string and a smaller final package size

**Review by insertish — APPROVED** [review]

**Pull request merged**